boards/sim/sim/sim/src/.gitignore

```diff
@@ -1,2 +1 @@
 /etctmp*
-!/etc/**
```

Comment:
I am conflicted on this one because shouldn't we allow anything in this folder?  I guess going forward we should just be careful when people force the inclusion of a file and instead have a specific git ignore exception for it.

Reply:
That change was part of #1373 to solve the *.d issue.  I don't know if we should allow anything in that folder, if you think we should I can bring it back.
> I guess going forward we should just be careful when people force the inclusion of a file and instead have a specific git ignore exception for it.

I agree with this.  But does this apply to this change?  I guess somehow it does since at the very beginning the idea was to bring the rcS file back.
But what I'm trying to do is the have a general way to include *.d folders since we may have more of them.

Reply:
@Ouss4 @btashton how about we change !*.d/ to !init.d/ or !etc/? since etc/ is packaged into romfs, files contained by etc can be any types that .gitignore is hard to anticipate.

Reply:
@Ouss4 @btashton how about we change !*.d/ to !init.d/ or !etc/? since etc/ is packaged into romfs, files contained by etc can be any types that .gitignore is hard to anticipate and then ignore correctly.

Reply:
The reason that I don't really like handling this at the top level gitignore is that is is really board configuration specific.  Do we add `home` `lib` `bin` etc..

I think I would be OK with adding the !*.d/ rule at the top because there really is no reason to ever exclude that, but opening up everything inside the folder I think we should be doing as needed and at the board level.

Reply:
> I think I would be OK with adding the !*.d/ rule at the top because there really is no reason to ever exclude that, but opening up everything inside the folder I think we should be doing as needed and at the board level.

The folder still ignores toplevel patterns.  So it's not completely open.